src/components/grid-aware/StatsBlock/StatsBlock.tsx

```diff
@@ -5,29 +5,44 @@ import * as s from "./StatsBlock.module.css";
 type StatCardProps = {
   number: string;
   statement: string;
+  theme?: string;
```

Comment:
Without the `theme` prop being optional, it requires each stat card to pass in a `theme` prop. Is there a better way of passing in the theme from `<StatsBlock>` to `<statCards>`?

```
<StatsBlock
        title="See what our volunteers have accomplished together."
        statCards={[
          {
            number: "9,000",
            statement:
              "At-risk community members can now access services and amenities",
            theme="white",
          },
          {
            number: "560",
            statement:
              "Bay Area volunteers have contributed to our cause since our founding in 2016",
            theme="white",
          },
          {
            number: "$2M",
            statement:
              "Worth of work provided by volunteers has been put towards our programs",
            theme="white",
          },
        ]}
        theme="white"
      />
```